apps/site/src/app/global.css

```
@@ -15,314 +12,313 @@
-        var(--color-background-default) 0%,
-        var(--color-background-ppg) 262.5%
-    );
-    transition: background 0.3s ease;
+  display: flex;
+  height: 300px;
+  cursor: pointer;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 16px;
+  z-index: 10;
+  border-radius: 9px;
```

Stylelint (17.9.0)

[error] Duplicate property "border-radius" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)